openfisca_us/variables/gov/irs/total_income_tax.py

```diff
@@ -0,0 +1,11 @@
+from openfisca_us.model_api import *
+
+
+class total_income_tax(Variable):
```

Comment:
(Separately we may want to rename `income_tax` to `federal_income_tax`)
```suggestion
class federal_state_income_tax(Variable):
```